### PR TITLE
Configure the edit URL

### PR DESCRIPTION
**Pull request by ggrossetie**
Replace HEAD by main

### CODE REVIEW

**Comment by ggrossetie** [comment]
@mraible could you please give it a try with this change?

File: site.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ content:
     - url: '.'
       branches: HEAD
       start_path: src/docs/asciidoc
+      edit_url: '{web_url}/edit/main/{path}'
```
Comment (mraible) [inline site.yml:10]: Should the `branches` still be `HEAD` or should it change to `main`? If we change it to `main`, is `edit_url` still needed?

Comment (ggrossetie) [reply site.yml:10]: That's correct but then you will probably need one playbook for local editing/preview that targets `HEAD` and a playbook for publishing that targets `main`

Comment (mraible) [reply site.yml:10]: What does `HEAD` mean in this context? I'm not sure I understand what it's used for.

Comment (ggrossetie) [reply site.yml:10]: It means the current local branch. It's useful for "deploy previews" on Netlify where you want to use the latest content from the current branch.
The documentation states:

> The value HEAD is equivalent to using the name of the current branch.

**Pull request merged**